ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -114,3 +114,40 @@ static_library('nnstreamer_decoder_image_segment',
   install: true,
   install_dir: nnstreamer_libdir
 )
+
+# protobuf decoder
+if get_option('enable-protobuf')
+  # Compile protobuf schema: nnstreamer.proto
+  pb_comp = find_program('protoc', required: false)
+
+  if not pb_comp.found()
+    error('protoc is not found')
+  endif
+
+  pb_gen = generator(pb_comp, output: ['@BASENAME@.pb.h', '@BASENAME@.pb.cc'], arguments : ['--proto_path=@CURRENT_SOURCE_DIR@', '--cpp_out=@BUILD_DIR@', '@INPUT@'])
+  pb_gen_src = pb_gen.process('nnstreamer.proto')
+
+  decoder_sub_protobuf_sources = [
+    'tensordec-protobuf.cc',
+  ]
```

Comment:
maybe... ```decoder_sub_protobuf_sources += pb_gen_src``` is more neat? (remove pb_gen_src from below)

Reply:
I updated it. Please see the very next line.